api/SVGGeometryElement.json

```diff
@@ -0,0 +1,344 @@
+{
+  "api": {
+    "SVGGeometryElement": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGGeometryElement",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "53"
+          },
+          "firefox_android": {
+            "version_added": "53"
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": null
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "supportOtherThanPath": {
+        "__compat": {
+          "description": "Support for elements other than <code>&lt;path&gt;</code>",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "53"
```

Comment:
Table here shows it as not being supported https://developer.mozilla.org/en-US/docs/Web/API/SVGGeometryElement however following the bug (https://bugzilla.mozilla.org/show_bug.cgi?id=1325320) in the note we find that it became supported in 61, I assume for both desktop and mobile versions. @Elchi3 should be able to confirm.

Reply:
Yeah, 61 is correct here. I updated that.